dev/contributing.rst

```diff
@@ -109,16 +109,23 @@ Go Specific
 Commits
 -------
 
-- The commit message subject should be a single short sentence
-  describing the change, starting with a capital letter but without
-  ending punctuation, and prefixed with the package name most affected
-  by the change.
+- Commits messages (and pull request titles) should follow the [conventional
+  commits](https://www.conventionalcommits.org/en/v1.0.0/) specification and
+  be in lower case.
+
+- We use a scope description in the commit message subject. This is the
+  component of Syncthing that the commit affects. For example, ``gui``,
+  ``protocol``, ``scanner``, ``upnp``, etc -- typically, the part after
+  ``lib/`` or ``cmd/`` in the package path. If the commit doesn't affect a
+  specific component, such as for changes to the build system or
+  documentation, the scope should be omitted. The same goes for changes
+  that affect many components which would be cumbersome to list.
 
 - Commits that resolve an existing issue must include the issue number
   as ``(fixes #123)`` at the end of the commit message subject. A correctly
   formatted commit message looks like this::
 
-    lib/dialer: Add env var to disable proxy fallback (fixes #3006)
+    feat(dialer): add env var to disable proxy fallback (fixes #3006)
```

Comment:
Thought we wanted to move the "fixes" mention from the subject to the message body?